Introduce ballerina-xslt module (#16387)
ballerina-platform/ballerina-lang: stdlib/xslt/src/main/ballerina/xslt/Module.md

## Module Overview

> This module provides a function to transforms the XML to another XML/HTML/plain text using XSL transformations.

Reviewer: ```suggestion
This module provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
```

## Samples

> ###Transform xml using a given xsl

Reviewer: ```suggestion
###Transform XML using a given XSL
```

> The following program depicts how the xsl transformation can be done for given single rooted xml, using a xsl.

Reviewer: ```suggestion
The following program depicts how the XSL transformation can be done for a given single-rooted XML using an XSL.
```
